src/views/pages/sign-in.ejs

```diff
@@ -6,30 +6,38 @@
 	<body>
 		<div class="wrapper">
       <div class="container">
-        <h1>З поверненням</h1>
+        <div class="sign-in-welcome-container">
+        <h1 class="sign-in-title">З поверненням!</h1>
```

Comment:
```suggestion
        <div class="welcome-container">
        <h2 class="heading">З поверненням!</h1>
```
remove sign-in part everywhere, as it's already sign in page and we're using specific sign-in styles